docs/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python
+
+COPY --chown=1000:1000 ./requirements.txt ./
+COPY --chown=1000:1000 ./run-documentation.sh ./
```

Comment:
```suggestion
ADD ./requirements.txt ./run-documentation.sh  /
```

Reply:
Done in 618a8892f93dd794a6000baa838503371ec1eb6d